openlibrary/i18n/validators/format_validator.py

```diff
@@ -0,0 +1,29 @@
+from babel.messages.catalog import TranslationError
```

Comment:
This class hierarchy feels like too much here. The fact that both subclasses (1) don't override the constructor, and (2) have only one method, makes it look like these are just functions in disguise as classes. Let's dissolve the base class, and have two functions in a single `validators.py` file: `validate_python_format(message, catalog)` and `validate_not_fuzzy(message, catalog)`.